### PR TITLE
refactor: replace superspawn with execa

### PR DESCRIPTION
**Pull request by erisu**
### Motivation and Context

https://github.com/apache/cordova/issues/16

### Description

Replace `superspawn` usage with `execa` 

### Testing

- `npm t`

### Checklist

- [x] I've run the tests to see all new and existing tests pass

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -22,6 +22,7 @@
   },
   "dependencies": {
     "cordova-common": "^3.1.0",
+    "execa": "^2.0.4",
```
Comment (timbru31) [inline package.json]: Any reason to use an old version of execa? (see https://github.com/sindresorhus/execa/releases/tag/v3.2.0)

**Review by timbru31 — DISMISSED** [review]

File: package.json (hunk 1 above)
Comment (raphinesse) [reply package.json]: Good catch

**Review by raphinesse — APPROVED** [review]

**Review by NiklasMerz — APPROVED** [review]
We should finally merge it.

File: package.json [hunk 2]
```diff
@@ -17,6 +17,7 @@
   "dependencies": {
     "@npmcli/arborist": "^6.1.5",
     "cordova-common": "^4.1.0",
+    "execa": "^5.1.1",
```
[the remark this answers is not in the document]
Comment (erisu) [reply package.json:20]: Execa 6.0.0 and higher is an [ESM module](https://github.com/sindresorhus/execa/blob/v6.0.0/package.json#L13) which we can't use yet.

[the remark this answers is not in the document]
Comment (NiklasMerz) [reply package.json:20]: I wasn't aware. Good that you are paying attention.

**Pull request merged**